config/initializers/log_tags.rb

```diff
@@ -0,0 +1,9 @@
+Rails.application.configure do
+
+  config.log_tags = [
+    lambda { |request| "origin=#{request.ip}" },
+    lambda { |_| "tid=#{Thread.current.object_id}" },
+    lambda { |request| "request_id=#{request.uuid}" }
```

Comment:
Is there a reason for the renaming of both this field and this file?

Reply:
what do you mean by renaming the field?

I renamed the file from `logging` to `log_tags` as it is more specific to what's going on in this file.